### PR TITLE
doc: Add information about Checkov CY-3949

### PR DESCRIPTION
**Pull request by machadoit**

### CODE REVIEW

File: docs/getting-started/supported-languages-and-tools.md [hunk 1]
```diff
@@ -262,6 +263,13 @@ The table below lists all programming languages that Codacy supports and the cor
       <td><a href="https://pmd.github.io/pmd/pmd_userdocs_cpd.html">PMD CPD</a></td>
       <td><a href="https://github.com/realm/SwiftLint">SwiftLint</a><a href="#swiftlint-complexity"><sup>2</sup></a></td>
     </tr>
+    <tr>
+      <td>Terraform</td>
+      <td><a href="https://github.com/codacy/codacy-checkov">Checkov</a></td>
```
Comment (machadoit) [inline docs/getting-started/supported-languages-and-tools.md:268]: @pauloribeiro-codacy does this repo uses some kind of Dictionary where I should add Checkov, or should I ignore it Codacy side?

Comment (machadoit) [reply docs/getting-started/supported-languages-and-tools.md:268]: @lolgab found it! thx! :D

Comment (pauloribeiro-codacy) [reply docs/getting-started/supported-languages-and-tools.md:268]: Beautiful. :+1:

**Review by pauloribeiro-codacy — APPROVED** [review]
It's perfect! :ok_hand: :shipit:

**Pull request merged**